src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListReportPackage.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using NuGet.Protocol;
+
+namespace NuGet.CommandLine.XPlat.ListPackage
+{
+    internal class ListReportPackage
+    {
+        internal string PackageId { get; set; }
```

Comment:
Seems like all of these are always set correct? 
This object is only created after everything has been calculated first.

Should we instead set through a constructor and have this open be immutable?

We can have multiple constructors too (I see in the tests, you don't always set everything for example).

I was primary looking at the json output renderer tests and it was more difficult to read as all the assignments were taking a lot of vertical space.

Reply:
Addressed.